spec/integration/recipes/local.rb

```diff
@@ -0,0 +1,10 @@
+package 'sl' do
+  version '3.03-17+b2'
+end
+
+######
+
+gem_package 'bundler' do
+  version '1.17.3'
+  options ['--no-document']
```

Comment:
New rubygems, which is installed in `ruby` docker image, does not have `--no-rdoc` and `--no-ri` options. So use `--no-document` option instead.

Note: Ubuntu trusty, which is used in the existing integration test, installs Ruby 1.9.3!!!!! Rubygems on Ruby 1.9.3 has `--no-rodc` and `--no-ri` options :upside_down_face: .